src/app/home/home.component.ts

```diff
@@ -8,8 +8,9 @@ import {PageMetaService} from '../page-meta.service';
 import {HighlightCard} from "./HighlightCard";
 import {environment} from "../../environments/environment";
 
-const CCOpenDate = new Date('2023-11-28T12:00:00+00:00');
-const CCCloseDate = new Date('2023-12-05T12:00:00+00:00')
+const CCCloseDate = new Date('2023-12-05T12:00:00+00:00');
+const Jan8th = new Date('2024-01-08T12:00:00+00:00');
+const Feb9th = new Date('2024-02-09T12:00:00+00:00');
```

Comment:
I'm not sure what times of day we want to use - I've put 12 noon in because that's what we've often done before, but would be good to confirm. @BethBigGive ?